terraform/projects/infra-jump-instance/main.tf

```diff
@@ -0,0 +1,103 @@
+## Variables
+
+variable "additional_tags" {
+  type        = "map"
+  description = "Stack specific tags to apply"
+  default     = {}
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "remote_state_bucket" {
+  type        = "string"
+  description = "S3 bucket we store our terraform state in"
+  default     = "ecs-monitoring"
+}
+
+variable "stack_name" {
+  type        = "string"
+  description = "Unique name for this collection of resources"
+  default     = "ecs-monitoring"
+}
+
+variable "jumpbox_cidrs" {
+  type        = "list"
+  description = "List of CIDRs which are able to access the jumpbox"
+  default     = ["0.0.0.0/0"]
+}
+
+## Providers
+
+terraform {
+  required_version = "= 0.11.7"
+
+  backend "s3" {
+    key = "infra-jump-instance.tfstate"
+  }
+}
+
+provider "aws" {
+  version = "~> 1.14.1"
+  region  = "${var.aws_region}"
+}
+
+provider "template" {
+  version = "~> 1.0.0"
+}
+
+## Data sources
+
+data "terraform_remote_state" "infra_networking" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.remote_state_bucket}"
+    key    = "infra-networking.tfstate"
+    region = "${var.aws_region}"
+  }
+}
+
+data "terraform_remote_state" "infra_security_groups" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.remote_state_bucket}"
+    key    = "infra-security-groups.tfstate"
+    region = "${var.aws_region}"
+  }
+}
+
+## Resources
+
+module "jumpbox" {
+  source          = "../../modules/jumpbox"
+  aws_region      = "${var.aws_region}"
+  subnet_id       = "${element(data.terraform_remote_state.infra_networking.public_subnets, 0)}"
+  stack_name      = "${var.stack_name}"
+  vpc_id          = "${data.terraform_remote_state.infra_networking.vpc_id}"
+  security_groups = ["${data.terraform_remote_state.infra_security_groups.monitoring_external_sg_id}"]
+  allowed_cidrs   = "${var.jumpbox_cidrs}"
```

Comment:
Are we allowing any IP address access to the jumpbox as opposed to just the GDS IPs here which would have been the default if we had not provided a value for `allowed_cidrs`? If so, is there a reason for that? I can't remember how cidr blocks work exactly so I might have misunderstood

Reply:
I'll check with @ejrowley on this